bundles/org.openhab.binding.hapero/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="hapero" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Hapero Binding</name>
+	<description>Binding for data aquisition from Hapero pellet stoves.</description>
```

Comment:
Please also provide `connection` - see https://next.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions

Reply:
That label is optional according to the link but i will add it.